jobs/terraform-deploy/terraform-deploy.groovy

```diff
@@ -46,8 +46,7 @@ terraform ${action} -var env=${environment_name} -input=false ${extraopts}
 # AWS
 cd ../aws
 . ~/.aws_credentials
-terraform ${action} -var env=${environment_name} -input=false ${extraopts} \
--var "aws_access_key=${aws_access_key}" -var "aws_secret_key=${aws_secret_key}"
+terraform ${action} -var env=${environment_name} -input=false ${extraopts}
```

Comment:
Good point. Although they are visible in the job, as opposed to hidden in the vault.

I'm not sure whether unifying them is that important. We could wrap them with a `Makefile` like we have for Ansible.